config/prow/config_knative.yaml

```diff
@@ -64,73 +64,55 @@ presubmits:
       dot-dev: true
       always_run: false
       optional: true
-      args:
-      - "--run-test"
-      - "./test/e2e-tests.sh --istio-version 1.3-latest --mesh &&
+      full-command: "./test/e2e-tests.sh --istio-version 1.3-latest --mesh &&
```

Comment:
This is an ugly hack (and I don't think it will work).

I think adding an additional script is a better solution, it'll basically just look like:

```
./e2e-tests.sh $@
./e2e-tests.sh --run-tests --skip-knative-setup $@
```

(actually, you seem to just be running the same test twice? I can't see what's different except the above options?)

Reply:
This is more like a test to see which one works for Prow, and I do agree with you that have a additional script is a better solution in this case, and it could live inside serving repo

Reply:
Indeed this won't work, as the successful second run will overwrite the results of the first run. Also, presubmit or not, they all work the same way so you can just have a separate job that you can start from command line for testing purposes, no need to merge a test PR.

Reply:
Apologies, this will actually run, but as presubmit it will miss the presubmit checks. I commented on 1746.